### DISCUSSION TITLE
Event gets created for different time

### DISCUSSION THREAD

**Discussion by meet1919** [kuzmoyev/google-calendar-simple-api#159] [General]
I am creating a timed event. The time which I pass, changes when event is created.

```python
title = 'Movie night'
start = (start_day/start_month/start_year)[st_hr:st_min]  # when printing this shows = 2023-01-06 15:00:00
end = (end_day/end_month/end_year)[en_hr:en_min] # 2023-01-06 17:00:00

event = Event(summary=title, start=start, end=end)
```

I can see that `Event` class has `timezone` set by default to `tzlocal.get_localzone_name()` 
I cant seem to understand why this isnt working properly

**Comment by kuzmoyev** [comment]
Could you please show how the `event.start` look like?

**Reply by meet1919** [reply]
It looks like this `2023-01-03 20:00:00+05:30`

which seems correct.

The event which I created from google calendar UI at 8pm the time show by `event.start.__str__()` when fetching all events is = `2023-01-04 20:00:00-08:00`

it is showing `-08:00` instead of `+05:30`

**Reply by kuzmoyev** [reply]
That means that the default timezone of the calendar is different from the timezone of your machine. You can specify timezone you want yourself with the `timezone` parameter of the `Event`

**Reply by meet1919** [reply]
Solved the issue by setting the timezone correctly.
